core/const.py

```diff
@@ -870,3 +870,171 @@ class ITLRegion(StrEnum):
         "RiskOwnerRole": "P{i}",
     },
 }
+
+# map place to fund type associated with it
+PLACE_TO_FUND_TYPE = {
+    "Heanor": {"Future_High_Street_Fund"},
```

Comment:
Pre-transformation validation for place_name/fund_type works by looking at an ```entered_value``` and seeing if it is in a set of ```expected_values``` because we don't want and there is no reason for these expected_values to be duplicated so this dictionary uses a set as well in order to be validated by the same function without altering the datatype.